openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -89,6 +89,25 @@
         benchmarks to remediate high, and moderate risk flaws within a
         customer defined period after a flaws discovery.
         */'
+
+- control_key: SI-2(4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: complete
+  narrative:
+    - key: a
+      text: |
+        Withdrawn
```

Comment:
```suggestion
        This control was withdrawn by NIST and incorporated into SI-2.
```

Reply:
We can't always assume the user will know why this control was withdrawn, where it went, or who withdrew it (eg "did Red Hat just decide this wasn't applicable?"). A slightly more verbose answer helps provide clarity.

Another example of "withdrawn control" is from AC-3 (1):
https://github.com/ComplianceAsCode/redhat/blob/master/openshift-container-platform-4/policies/AC-Access_Control/component.yaml#L279#L286

`````
- control_key: AC-3 (1)
  standard_key: NIST-800-53
  covered_by: []
  implementation_status: not applicable
  narrative:
    - text: |-
        This control was withdrawn by NIST, and incorporated into
        AC-6
`````